.azure-pipelines/azure-pipelines-merge.yml

```diff
@@ -192,7 +192,7 @@ jobs:
     vmImage: 'macOS-10.15'
 
   steps:
-  - script: git checkout release-0.21.0
+  - script: git checkout release-0.22.0
```

Comment:
make sure to create a branch with name as `release-0.22.0` before merging this PR

Reply:
Done!

Ref: https://github.com/Azure/azure-devops-cli-extension/tree/release-0.22.0
Confirmation: Created before merge